build/template-pack-and-sign-packages.yaml

```diff
@@ -52,8 +52,8 @@ steps:
     targetType: 'inline'
     script: |
       dotnet nuget locals all --clear 
-      dotnet workload install maui --source https://aka.ms/dotnet6/nuget/index.json --source https://api.nuget.org/v3/index.json
-      dotnet workload install android ios maccatalyst tvos macos maui wasm-tools --source https://aka.ms/dotnet6/nuget/index.json --source https://api.nuget.org/v3/index.json
+      dotnet workload install maui --source https://api.nuget.org/v3/index.json
+      dotnet workload install android ios maccatalyst tvos macos maui wasm-tools --source https://api.nuget.org/v3/index.json
```

Comment:
nit: we can also probably remove maccatalyst,  tvos, wasm tools? (and from ios tests yaml too)